qa/suites/orch/cephadm/workunits/task/test_host_drain.yaml

```diff
@@ -46,6 +46,16 @@ tasks:
           printf "Expected to see $HOST_C in CRUSH map. Saw:\n\n$CRUSH_MAP"
           exit 1
         fi
+        # One last thing to worry about before removing the host
+        # is that the teuthology test tends to put the explicit
+        # hostnames in the placement for the mon service.
+        # We want to make sure we can remove without providing
+        # --force and there is a check for the host being removed
+        # being listed explicitly in the placements. Therefore,
+        # we should remove it from the mon placement.
+        ceph orch ls mon --export > mon.yaml
+        sed /"$HOST_C"/d mon.yaml > mon_adjusted.yaml
+        ceph orch apply -i mon_adjusted.yaml
```

Comment:
did this in the wrong spot. Need to be before the drain command, not the `host rm` command
```
Error EINVAL: Host smithi069 was found explicitly listed in the placements of services
```
https://pulpito.ceph.com/adking-2023-11-08_01:34:19-orch:cephadm-wip-adk-testing-2023-11-07-1631-distro-default-smithi/7451644/